### ISSUE TITLE
Support TIMESTAMP_NS, TIMESTAMP_MS & TIMESTAMP_S types

### ISSUE THREAD

**Issue by JelteF** [duckdb/pg_duckdb#283]
DuckDB has timestamp types with different precisions. We should support conversion from/to those types. `TIMESTAMP_NS` is probably the most challenging, since postgres does not have a native timestamp type with nanosecond precision. It seems better to truncate/round the nanoseconds to microseconds though than to not support them at all.

Related to #228, which tracks adding `TIMESTAMP WITH TIME ZONE`.

**Comment by destrex271** [comment]
Hi @JelteF , Picking this up.

**Comment by JelteF** [comment]
@destrex271 are you still expecting to submit a PR for this?

**Comment by destrex271** [comment]
> @destrex271 are you still expecting to submit a PR for this? 

Yes, just got stuck in some other stuff. Will try to raise a pr asap

**Comment by destrex271** [comment]
For conversion from lets say `TIMESTAMP_MS` to `TIMESTAMP` we can abstract it in the `TIMESTAMP` type in postgres, but to convert it back to the original duckdb type, it might get a little hard to manage with this approach. 

Should we create a new type/datum like TIMESTAMP_MS for pg as well?
